src/libcore/fmt/mod.rs

```diff
@@ -609,10 +609,15 @@ pub trait Debug {
 /// println!("The origin is: {}", origin);
 /// ```
 #[rustc_on_unimplemented(
+    on(
+        _Self="std::path::Path",
+        label="`{Self}` cannot be formatted with the default formatter, call `.display()` on it",
+        note="you need to call `.display()` or `.to_string_lossy()` for safely printing paths as \
+              they may contain non-Unicode data"
```

Comment:
I was given to understand that `Display` needed to be UTF-8 specifically (this being a more stringent constraint than "Unicode")?

Reply:
What should be the proposed wording change? UTF-16 and UTF-32 encoded strings can always be safely converted to UTF-8 for printing, so they do/would implement `Display` always, right?

Reply:
Withdrawn.